aks-kubesphere-linux/azuredeploy.json

```diff
@@ -0,0 +1,506 @@
+            },
+            "minValue": 0,
+            "maxValue": 1023
+        },
+        "dnsPrefix": {
+            "type": "string",
+            "defaultValue": "AKS-KubeSphere-dns",
+            "metadata": {
+                "description": "Optional DNS prefix to use with hosted Kubernetes API server FQDN."
+            }
+        },
+        "kubernetesVersion": {
+            "type": "string",
+            "allowedValues": [
+                "1.17.13",
+                "1.17.16",
+                "1.18.10",
+                "1.18.14",
+                "1.19.6",
+                "1.19.7"
+            ],
+            "defaultValue": "1.18.14",
+            "metadata": {
+                "description": "The version of Kubernetes."
+            }
+        },
+        "MasterNodeCount": {
+            "type": "int",
+            "defaultValue": 3,
+            "allowedValues": [
+                1,
+                3,
+                5
+            ],
+            "metadata": {
+                "description": "The number of Master node."
+            }
+        },
+        "WorkerNodeCount": {
+            "type": "int",
+            "defaultValue": 3,
+            "allowedValues": [
+                1,
+                2,
+                3,
+                4,
+                5,
+                6
+            ],
+            "metadata": {
+                "description": "The number of Worker node."
+            }
+        },
+        "enablePrivateCluster": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Enable private network access to the kubernetes cluster."
+            }
+        },
+        "ClientResourceName": {
+            "type": "string",
+            "defaultValue": "aksclient",
+            "metadata": {
+                "description": "The name of AKS client node"
+            }
+        },
+        "ClientNodeNetworkInterfaceName": {
+            "type": "string",
+            "defaultValue": "ksnode-nic",
+            "metadata": {
+                "description": "The name of new or existing client node network interface."
+            }
+        },
+        "ClientNodeEnableAcceleratedNetworking": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Enable client node network accelerated."
+            }
+        },
+        "ClientosDiskType": {
+            "type": "string",
+            "allowedValues": [
+                "StandardSSD_LRS",
+                "Premium_LRS"
+            ],
+            "defaultValue": "StandardSSD_LRS",
+            "metadata": {
+                "description": "Client os Disk type"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "aksuser",
```

Comment:
remove defaultValue

Reply:
This is my fault. The defaultValue of adminUser and password should not be set. This is not standard. I will check all files carefully.